installer_scripts/install-yb-voyager-debezium

```diff
@@ -65,6 +65,48 @@ main() {
 # COMMON
 #=============================================================================
 
+# https://stackoverflow.com/a/4025065
```

Comment:
Can you add a comment stating `vercomp` returns 0 for equal 1 for var1>var2 etc.

Reply:
done